lib/core/presentation/pages/event/event_control_panel_page/sub_pages/event_reward_setting_page/sub_pages/event_create_reward_page/widgets/create_reward_basic_info_form.dart

```diff
@@ -77,7 +85,9 @@ class _CreateRewardBasicInfoFormState extends State<CreateRewardBasicInfoForm> {
               trailing: SizedBox(
                 width: 60.w,
                 child: LemonTextField(
-                  initialText: "1",
+                  initialText:
+                      modifyRewardBloc.initialReward?.limitPer.toString() ??
```

Comment:
just wonder if reward can be unlimited ?

Reply:
Right now it can't, even on web, we can enhance this later if requirement required this unlimited 👍